### ISSUE TITLE
Consider new behavioral phenotype terms for Ohyama 2015

### ISSUE THREAD

**Issue by dosumis** [FlyBase/flybase-controlled-vocabulary#10]

**Comment by dosumis** [comment]
A range of escape behaviors are described in papers referenced from this one:

"Larvae have... an array of distinct escape behaviours available for responding to threats, such as turning, digging, fast crawling and rolling 27,28,31–35 The most vigorous escape sequence consists of two locomotion modes: rolling followed by fast crawling 28,32."

As a follow-up to this ticket, we should probably investigate adding a wider range of larval movement phenotypes.

**Comment by aoifelarkin** [comment]
Just retrohacked this paper and it actually has been phen-curated (by me, though I have no memory of it!). Possibly your query is looking only for CV statements attached to the FBrf? Since this paper looks at behavior phenotypes that result from activating neurons - ie a GAL4 or LexA driver driving TrpA1 - we don't actually attach CV terms in these cases (not sure how much you know about this issue?). So you can see free text statements describing phenotypic consequences of activating the GAL4 under the phenotypic free text section (phenotypic data>detailed description) of the GAL4 (eg http://flybase.org/reports/FBal0254537.html).

We should still try and come up with potential new CV terms for this paper, but wanted to note that CV terms are not currently attached to GAL4s so we'll need to come up with a solution for that first (something that has been discussed previously at FlyBase, hence the introduction of the pheno_anat flag, see below). This will apply to any papers that activate/silence populations of neurons, such as FlyBowl. 

At some point since this paper, we introduced the pheno_anat flag to add to these kinds of papers (copied the details of that flag from the curation manual below), so that in future we could come up with a plan to deal with them more effectively. Looks like this paper also records activity of neurons (GCaMP), which is something we don't even capture in free text.

pheno_anat
The publication describes experiments that use genetic manipulations as a tool to activate/inhibit/kill a cell, to try to identify what biological processes that particular cell is normally involved in (i.e. they are not really concerned with the biology of the particular gene/gene product being used to affect the cell, they are just using it as a tool).  These manipulations may either be used to generate a phenotype in an otherwise wild-type background, or they may be used to study the interaction between activating/inihibiting/killing a particular cell and a phenotype caused by mutations in another gene. (Note, that at present we are not capturing this information because the way it ends up being reported is not very helpful to users, so this flag is acting as an easy way for us to identify these papers in the future, once we have implemented changes which solve this issue).

**Comment by dosumis** [comment]
> We should still try and come up with potential new CV terms for this paper, but wanted to note that CV terms are not currently attached to GAL4s so we'll need to come up with a solution for that first.

I didn't realise these are not being captured!  These are the phenotypes we are most interested in for VFB so we need to find a way to change this ASAP and come up with a targeted/prioritised retrofit strategy.   We should also discuss whether it is worth capturing anything from GCAMP experiments.  Perhaps some combination of neurophysiology defective + cell type could work here.  Need to work on a use case query and think about whether we have resources before we decide.

I'll be in Cambridge next Wed, perhaps we could discuss then.

**Comment by aoifelarkin** [comment]
Ah, thought you were aware! Yes, would be good to especially have a chat with Gillian about it as she's the most knowledgeable. Needing it for VFB (along with having some dedicated man-hours for it!) will be a good impetus to get it done.

**Comment by dosumis** [comment]
Good that we have free text though.  Definitely worth displaying to our users and looks like it could make retrofitting practical.

**Comment by temjones** [comment]
Just FYI, I think we decided as of July 2016 that instead of adding free text for activation/inhibition of neurons, we would ONLY add the pheno_anat flag. These papers will still be easily identifiable for a retrofit, but for papers curated since then, there won't be free text describing those types of experiments.

**Comment by aoifelarkin** [comment]
Yep - the pheno_anat flagged list of papers is part of what we pitched I would work on (thin/phen curation) in our VFB curation plans proposal (think Gillian gave you a copy and we discussed it briefly when we met for our training).

**Comment by dosumis** [comment]
Pasting from Slack: 

> tamsin [3:33 PM] 
> Hello - looking at Ohyama 2015 paper and potential behavioral phenotypes… Struggling a bit to think about how these might work, since activating certain circuits is provoking “normal” behavior. The phenotype ontology terms are largely based on defects in biological processes. So I am just not quite sure how this would fit into existing phenotype curation. Agree that it’s useful to be able to record it, but it’s quite different to typical phenotype curation (edited to correct typo) (edited)

> Aoife [3:35 PM] 
> Was chatting to Tamsin about this earlier and she brought this up, and I agree. Seems slightly odd or clumsy to attach CV terms (as we’ve been using them) to ‘wild-type’ phenotypes that particular circuits are responsible for.

I think we can still treat them as phenotypes in that they are behaviours induced when they wouldn't normally be. 

e.g. behaviour: wing flicking
phenotype induced by some genotype + heat shock: increased wing flicking

We have standard patterns to record this in phenotype ontologies.  Take a look at any of the phenotype terms in FBcv with 'increased' in the name.

With sufficient evidence, we might want to assert in the ontology that a neuron type is "'capable of part of' some 'wing flicking'".  Just as for a gene annotation with a GO term, you need loss of function evidence and strong evidence that the effect is specific to the neuron.  e.g. a combination of very specific targeting + GCAMP evidence that shows activation of the neuron during activation of the behaviour.   (FlyBowl is all activation not loss of function so should be only recorded as phenotype)

**Comment by aoifelarkin** [comment]
Had a quick look and found a few other papers that also looked at larval escape behavior, documenting them below for more information and also so that they can be curated at the same time as Ohyama Nature 2015 (**FBrf0228257**) (they may also contain useful follow up references). All the papers mention rolling behavior, and a couple also measure speed of crawling - these could potentially be the most reliable/consistent CV terms to use.

Ohyama et al., 2013, PLoS ONE (**FBrf0222494**) (paper has only been skimmed); "Noxious mechanical and thermal stimuli can evoke sideways rolling, a stereotyped escape response [9]. Two mechanical stimuli, touching and vibration, induce head retraction and head casting [10–12]."
Yoshino et al., 2017, Curr. Biol. (**FBrf0236440**) (no curation on this yet): "optogenetic activation of C4 da neurons is sufficient to provoke corkscrew-like rolling behavior similar to what is observed when larvae receive noxious stimuli, such as high temperature or harsh mechanical stimulation [10–12]"; they also seem to use 'rolling' and increases in crawling speed as measures, and have a nice movie of rolling behavior in the [supplement](http://www.sciencedirect.com/science/article/pii/S0960982217308059?via%3Dihub).
Oswald et al., 2011, Fly (**FBrf0217956**) (paper has only been skimmed); "We describe here a robust behavioral escape response from larvae to rising temperature." "In addition to the corkscrew roll escape response, we observed another aspect to the behavior preceding the roll where the larva thrashes the mouthparts laterally once to each side (Fig. 1A and Sup. Movie S1). This response invariably precedes the cork- screw roll."

**Comment by dosumis** [comment]
How do we name the phenotype when escape response is activated in the absence of stimulus.

ectopic escape response ?
induced escape response | conditional ts (need to add conditional optogenetic)

'increased escape response' feels wrong as doesn't occur much without stimulus.

**Comment by dosumis** [comment]
- [ ] Write referenced textual description of main larval locomotor modes - those of relevance to Ohyama paper are highest priority @aoifelarkin

**Comment by aoifelarkin** [comment]
In thinking about this and reading up some of the papers that were referenced by Ohyama Nature 2015 (FBrf0228257), along with FBrf0222494/FBrf0236440/FBrf0217956 mentioned in my comment above, I've seen some of these escape behaviors (including rolling sometimes mentioned in free text) tagged with some existing CV terms: such as pain response defective, or stress response defective, or behavior defective (for example check out phenotypic class terms and descriptions under http://beta.flybase.org/reports/FBal0281084.html). Presume some retrofitting would be required if a 'rolling' type CV term were to be added (and often free text is only listed under the UAS, not the GAL4).

'pain response defective' (a child term of stress response defective) ; the definition of this in FBcv is: Phenotype that is a defect in response to pain (GO:0048265). 'response to pain' is defined as: 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a pain stimulus. Pain stimuli cause activation of nociceptors, peripheral receptors for pain, include receptors which are sensitive to painful mechanical stimuli, extreme heat or cold, and chemical stimuli.' 

Possibly the escape rolling type behavior might fit under this CV term already (given that it is a change in movement in response to mechanical stimulus). Or if you wanted more detailed locomotor-escape CV terms, you could add the rolling escape behavior as an extra CV term under here?

FYI FBrf0223267 (http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0078704) has a nice movie (movie s4) of an escape roll in response to a parasitoid wasp, as well as a nice figure depicting larval behaviors in response to attack (fig 1). 

[FBrf0224808](http://flybase.org/reports/FBrf0224808.html) is an uncurated paper which has a lot of data ("We optogenetically activated 1054 identified neuron lines in Drosophila larvae and tracked the behavioral responses from 37,780 animals") in which they identify "29 discrete, statistically distinguishable, observer-unbiased behavioral phenotypes" - these also have nice videos in the supplemental data and might be useful as general larval locomotor categories (the phenotypes include some 'escape' subtypes).

Potential definition for a 'rolling behavior' type CV term: Escape (corkscrew-like) rolling behavior in larvae can occur in threatening situations such as strong noxious stimulation or predator attack (FBrf0200914, FBrf0222494, FBrf0223267, FBrf0224808). A combination of mechanosensory and nociceptive cues increases the likelihood of rolling (FBrf0228257).
